libs/configurable-product/guides/driver.md

```diff
@@ -0,0 +1,5 @@
+# Driver
+The `@daffodil/configurable-product` does not provide its own driver. Instead, this package uses injector functions provided by the `@daffodil/product` package to extend the graphql product fragment and the product transformers via a provided module.
```

Comment:
This will only be true for GraphQL drivers. And you actually don't use the injector functions anymore. I would say something like: ```Instead, this package hooks into extension mechanisms exposed by the `@daffodil/product` package ...```

Reply:
changed